strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/main/java/org/carlspring/strongbox/config/NugetLayoutProviderConfig.java

```diff
@@ -22,12 +22,10 @@
 
 @Configuration
 @ComponentScan({ "org.carlspring.strongbox.configuration",
-                 "org.carlspring.strongbox.event",
```

Comment:
(Just for my better understanding) : You've removed this, because it's handled via Spring's Event API, right?

Reply:
yes, you are right